Winapp2.ini

```diff
@@ -19211,16 +19093,6 @@ Default=False
 FileKey1=%CommonAppData%\Xerox\PrinterDriver|*.tmp|RECURSE
 FileKey2=%LocalAppData%\VirtualStore\ProgramData\Xerox\PrinterDriver|*.tmp|RECURSE
 
-[Xfire *]
-Section=Games
-Detect=HKLM\Software\Xfire
-Default=False
-Warning=Selecting this will delete any screen shots and videos you have taken, make sure you upload or move any you want to keep so they're not deleted.
-FileKey1=%CommonAppData%\Xfire|*.bak
-FileKey2=%CommonAppData%\xfire\videos|*.*
-FileKey3=%LocalAppData%\VirtualStore\ProgramData\Xfire|*.bak
-FileKey4=%LocalAppData%\VirtualStore\ProgramData\xfire\videos|*.*
-
 [Xfire Updates *]

```

Comment:
We can probably remove any xfire entries as the servers shut down in 2016

Reply:
Thanks for the information, another non-functional messenger! I will move both entries to the Archived entries.ini.